app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -440,17 +414,25 @@ const ContentForm: React.FC<IContentFormProps> = ({
                                   if (e.target.value.indexOf('_')) {
                                     e.target.value = e.target.value.replaceAll('_', '0');
                                   }
-                                  const date = getHours(e);
+                                  const date = setTime(props.values.publishedOn, e.target.value);
                                   if (!!date) {
+                                    props.setFieldValue(
```

Comment:
This is blatantly obvious...  Set the property that is used to save the `content.publishedOn` information...